web/src/components/vis/VisTile.vue

```diff
@@ -11,28 +11,61 @@ export default {
     },
   },
 
+  data() {
+    return {
+      scale: 1,
+    };
+  },
+
   computed: {
-    hasControls() {
-      return !!this.$slots.controls;
+    smallEnough() {
+      return this.scale <= 0.5;
+    },
+    largeEnough() {
```

Comment:
Prefer better names.  perhaps `isMaxScale` and `isMinScale`.

Reply:
those methods just exists because of limitation of pug